package.json

```diff
@@ -42,7 +45,10 @@
     "request": "^2.88.2"
   },
   "devDependencies": {
+    "@types/auto-launch": "^5.0.1",
+    "@types/counterpart": "^0.18.1",
     "asar": "^2.0.1",
+    "electron": "12.0.11",
```

Comment:
Was this intentional? I guess it's fine as a devDependency if necessary

Reply:
Yes, the typescript types are contained within. @types/electron is no longer to be used according to the docs.

Reply:
See https://www.npmjs.com/package/@types/electron